### ISSUE TITLE
Tridactyl breaks opening New Window by Firefox

### ISSUE THREAD

**Issue by Massimo-B** [tridactyl/tridactyl#4146]
firefox-91.7.0 on Linux
Tridactyl: Beta 1.22.0pre6186

Hi, I got Tridactyl working so far. :winopen and :tabopen are working. However using CTRL+N to open a new window by Firefox, the window opens, but is getting closed right after appearing.
I wouldn't mind about this detail, but it also breaks opening new windows by running a firefox command. Usually I do that for starting Firefox initially or if running to start a new window. What is broken and how can I solve it?

**Comment by bovine3dom** [comment]
I've never heard of this issue before, perhaps there's some conflict with another extension. Could you try disabling any other extensions and trying again?

**Comment by Massimo-B** [comment]
I tried all addon combinations, reduced to the minimum set of addons:

- boostrap loader
- Tab Mix Plus
- Tridactyl: Beta

Having _Tridactyl_ alone or _Tab Mix Plus_ alone works, but the combination breaks CTRL+N.

As Newtab setting in Tab Mix Plus I followed
https://github.com/tridactyl/tridactyl/issues/4145#issuecomment-1070834502
and have left `New Tab Page, Address` in Tab Mix Plus empty.
:newtab and :windowopen are working fine.
